stage1_fly/stage1_fly.mk

```diff
@@ -9,6 +9,8 @@ $(call setup-stamp-file,FLY_STAMP,aci-build)
 
 $(call inc-many,$(foreach sd,$(FLY_SUBDIRS),$(sd)/$(sd).mk))
 
+$(call inc-one,../stage1/stop/stop_fly.mk)
```

Comment:
It's true, but it still fits to the stage1 nspawn entrypoint.  I think we should consider it with greater directories reorganization (I agree with https://github.com/coreos/rkt/pull/2438#issuecomment-211848536)